src/app/modules/item/pages/item-by-id/item-by-id.component.html

```diff
@@ -4,11 +4,17 @@
 <alg-loading *ngIf="state.isFetching"></alg-loading>
```

Comment:
Although the ability to refresh was there before, I think it can be removed since it doesn't make sense to refresh missing or disallowed content

Reply:
Fixed